### PR TITLE
add Access-Controller-Allow-Origin header to webpack devServer

### PR DESCRIPTION
**Pull request by ktzhang**
https://github.com/quip/issues/issues/17121

Note: Will need to update package.json's quip-apps-webpack-config to use the latest version for each of the examples

### CODE REVIEW

**Review by jesseditson — COMMENTED** [review]
LGTM!

**Review by jesseditson — APPROVED** [review]

File: examples/salesforce-lightning-design-system/webpack.config.js [hunk 1]
```diff
@@ -58,5 +58,8 @@ module.exports = {
         contentBase: path.resolve(__dirname),
         port: 8888,
         inline: false,
+        headers: {
+            "Access-Control-Allow-Origin": "*",
+        },
```
Comment (jesseditson) [inline examples/salesforce-lightning-design-system/webpack.config.js:63]: This actually needs to change in the quip repo, stuff in `examples` is periodically overwritten from there.

**Review by jesseditson — CHANGES_REQUESTED** [review]

File: examples/salesforce-lightning-design-system/webpack.config.js (hunk 1 above)
Comment (ktzhang) [reply examples/salesforce-lightning-design-system/webpack.config.js:63]: @jesseditson  changed it in the quip repo in https://github.com/quip/quip/pull/11342
should I still change it here too? what's the process of copying the examples over?

Comment (jesseditson) [reply examples/salesforce-lightning-design-system/webpack.config.js:63]: You can run scripts/sync_example_live_apps.sh to update these - you can pass the path to this repo as an argument and commit the changes directly to master (or PRing them is fine too)

**Comment by ktzhang** [comment]
Will update the examples in a future PR

**Review by jesseditson — APPROVED** [review]
LGTM!

**Pull request merged**